src/main/java/com/eharmony/pho/query/criterion/Ordering.java

```diff
@@ -1,38 +1,100 @@
 package com.eharmony.pho.query.criterion;
 
+
 /**
  * Ordering criterion
+ * This defines the order that results will be returned in.
  */
 public class Ordering implements Criterion, WithProperty {
 
     public static enum Order {
         ASCENDING, DESCENDING
     }
+    
+    /**
+     * When property name is <tt>null</tt> should those values be put first or last
+     */
+    public static enum NullOrdering {
+        FIRST, LAST
+    }
 
     private final String propertyName;
     private final Order order;
+    private final NullOrdering nullOrdering;
 
+    /**
+     * Create an ordering to order results by the given property in ascending order with nulls first
+     * @param propertyName the name of the property to order by
+     * @return the ordering
+     */
     public static Ordering asc(String propertyName) {
-        return new Ordering(propertyName, Order.ASCENDING);
+        return new Ordering(propertyName, Order.ASCENDING, NullOrdering.FIRST);
```

Comment:
Do you think we should provide another method to take the NullOrdering and default it to FIRST?